src/features/tag/components/TagCreateItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { cn } from "@/src/utils/tailwind";
+import { CommandItem } from "cmdk";
+
+type TagItemCreateProps = {
+  inputValue: string;
+  options: string[];
+  onSelect: () => void;
+};
+
+const TagItemCreate = ({
```

Comment:
can we use `CommandItemCreate` here?

Reply:
CommandItemCreate has currently prompt name hard coded and uses different css.
We could look into generalising the CommandItemCreate component and then reuse.